rules/macos/persistence_login_logout_hooks_defaults.toml

```
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/12/07"
+maturity = "production"
+updated_date = "2020/12/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of the Defaults command to install a LoginHook Persistence. MacOS LoginHooks allow a script to be
+automatically executed at login and/or logout.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Persistence via Login and/or Logout Hooks"
```

```suggestion
name = "Persistence via Login or Logout Hook"
```
there can only be one Login hook at a time so `and` wasn't needed